raster-foundry/raster-foundry
# Project layers share page

## Overview
Add a project layers share page
* Show layers for the shared project
* Show analyses for the shared project

### Checklist

- ~Styleguide updated, if necessary~
- [x] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated [here](https://github.com/raster-foundry/raster-foundry-api-spec/pull/96)
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Demo
![image](https://user-images.githubusercontent.com/4392704/54393034-805ed400-467f-11e9-9b95-0e8fc0e92b16.png)
![image](https://user-images.githubusercontent.com/4392704/54393054-8fde1d00-467f-11e9-844f-e20ca7e585b1.png)

### Notes
All layers and analyses on projects shared by mapToken are visible.
Visibility for layers / analyses on the api for public projects may not be fully supported, but I've spent enough points at this point and it covers the primary use case of sharing via mapToken


## Testing Instructions

 * Open the share page of a project with a mapToken in incognito
* add /v2/ before `share` in the URL and verify that the page loads without being logged in
* verify that layers can be toggled, zoomed to, and urls copied
* verify that the analyses list loads, and the same actions can be taken on them

Closes https://github.com/raster-foundry/raster-foundry/issues/4544
Closes #4763
